src/test/spec/unified_runner/mod.rs

```diff
@@ -91,15 +75,20 @@ pub async fn run_unified_format_test_filtered(
             }
         }
         if !can_run_on {
-            log_uncaptured("Client topology not compatible with test");
+            log_uncaptured(format!(
```

Comment:
the various logging changes from here down are not strictly related, but I was having a hard time reading the output to figure out if the new test files were being run properly or not so I decided to clarify the log statements a bit